src/data.js

```diff
@@ -0,0 +1,52 @@
+var sides = [
+'Miso Glazed Carrots',
```

Comment:
As a personal preference, I much prefer having this sort of data in a separate file rather than in the main.js.